### PR TITLE
Revert "last call" status

### PR DESCRIPTION
**Pull request by Arachnid**
These EIPs have been in "last call" for more than two weeks, with no updates from their author justifying moving to final, or back to draft. As a result, this PR moves them back to draft state.

### CODE REVIEW

**Comment by Arachnid** [comment]
CC @jbaylina @jacquesd @sorpaas

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - Trying to change EIP 1283 state from Last Call to Draft
 - Trying to change EIP 820 state from Last Call to Draft
 - EIP 1283 requires approval from one of (@sorpaas)
 - EIP 820 requires approval from one of (jacques@dafflon.tech, @jbaylina)

**Comment by sorpaas** [comment]
@Arachnid At least for EIP-1283, there has been justification to move it to "Accepted" a long time ago: https://github.com/ethereum/EIPs/pull/1565

**Comment by Arachnid** [comment]
@sorpaas I missed that, thank you. Removing that PR from this.

File: EIPS/eip-820.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ eip: 820
 title: Pseudo-introspection Registry Contract
 author: Jordi Baylina <jordi@baylina.cat>, Jacques Dafflon <jacques@dafflon.tech>
 discussions-to: https://github.com/ethereum/EIPs/issues/820
-status: Last Call
+status: Draft
```
Comment (sea51930) [inline EIPS/eip-820.md]:
```suggestion
status: Draft
```

**Comment by nicksavers** [comment]
There has been no further objection against this PR for over 2 weeks.

**Pull request merged**